### ISSUE TITLE
Multiple clozes in one block with individual stats tracking

### ISSUE THREAD

**Issue by SplitZadar** [akrivka/roamsr#27]
During review only one of the block's clozes should be hidden (or more if merged). Each cloze deletion in the block should have its own SR stats.

**Comment by hungriesthippo** [comment]
what do you think about a button that would turn that into multiple blocks?
since history is based on block refs, having multiple histories for a single block would require pretty significant changes I think

**Comment by SplitZadar** [comment]
Yes, the button would be great. In fact, that's my current workaround but I copy the prompt variations manually. Also, is there a chance to add this functionality as a block's context menu item?
